experimental/regression_suite/shark-test-suite-models/sd3/test_clip.py

```diff
@@ -0,0 +1,167 @@
+# Copyright 2024 The IREE Authors
+#
+# Licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+import pytest
+from ireers_tools import *
+import os
+from conftest import VmfbManager
+
+rocm_chip = os.getenv("ROCM_CHIP", default="gfx90a")
+
+###############################################################################
+# Fixtures
+###############################################################################
+
+sd3_clip_inference_input_0 = fetch_source_fixture(
+    "https://sharkpublic.blob.core.windows.net/sharkpublic/sai/sd3-prompt-encoder/inference_input.0.bin",
+    group="sd3_clip",
+)
+
+sd3_clip_inference_input_1 = fetch_source_fixture(
+    "https://sharkpublic.blob.core.windows.net/sharkpublic/sai/sd3-prompt-encoder/inference_input.1.bin",
+    group="sd3_clip",
+)
```

Comment:
Fine for now, but we should zip these (https://github.com/nod-ai/SHARK-TestSuite/issues/285) or use an array somehow here. This is a lot of boilerplate to download and use individual loose files.